README.md

```diff
@@ -6,50 +6,51 @@ Publication Light at any extension of the model.
+| Kienzler: Bike and Ride                                        | bike    | pull        | `kienzler_bike_and_ride` | yes      |
+| Kienzler: Karlsruhe                                            | bike    | pull        | `kienzler_karlruhe`      | yes      |
+| Kienzler: Neckarsulm                                           | bike    | pull        | `kienzler_neckarsulm`    | yes      |
+| Kienzler: Offenburg                                            | bike    | pull        | `kienzler_offenburg`     | yes      |
+| Kienzler: RadSafe                                              | bike    | pull        | `kienzler_rad_safe`      | yes      |
+| Kienzler: Stuttgart                                            | bike    | pull        | `kienzler_stuttgart`     | yes      |
+| Kienzler: VRN                                                  | bike    | pull        | `kienzler_vrn`           | yes      |
+| Konstanz                                                       | car     | pull        | `konstanz`               | yes      |
+| Stadt Konstanz: Fahrrad-Abstellanlagen                         | bike    | push        | `konstanz_bike`          | no       |
+| Stadt Mannheim                                                 | car     | push (json) | `mannheim`               | yes      |
+| Stadt Neckarsulm: PKW-Parkplätze                               | car     | pull        | `neckarsulm`             | no       |
+| Stadt Neckarsulm: Fahrrad-Abstellanlagen                       | bike    | pull        | `neckarsulm_bike`        | no       |
+| Open-Data-Plattform öV Schweiz                                 | car     | pull (json) | `opendata_swiss`         | no       |
+| P + M Baden-Württemberg                                        | car     | pull        | `p_m_bw`                 | yes      |
+| Baden-Württemberg: Parken und Mitfahren                        | car     | push (xlsx) | `pum_bw`                 | no       |
+| RadVIS Baden-Württemberg (experimental)                        | bike    | pull        | `radvis_bw`              | no       |
+| Parkraumgesellschaft Baden-Württemberg                         | car     | pull        | `pbw`                    | yes      |
+| Stadt Pforzheim                                                | car     | push (csv)  | `pforzheim`              | no       |
+| Stadt Reutlingen: PKW-Parkplätze                               | car     | push (csv)  | `reutlingen`             | no       |
+| Stadt Reutlingen: Fahrrad-Abstellanlagen                       | bike    | push (csv)  | `reutlingen_bike`        | no       |
+| Stadt Stuttgart                                                | car     | push (json) | `stuttgart`              | yes      |
+| Stadt Ulm                                                      | car     | pull        | `ulm`                    | yes      |
+| Velobrix                                                       | bike    | pull (json) | `velobrix`               | yes      |
```

Comment:
At pull converters, it does not really matter which data type it is. The data type is there at the push converters because the endpoints are different depending on the data type.